modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
```

Comment:
This heading doesn't seem to fit. Aren't we replacing/invalidating the kubeconfig? Isn't that the point of the procedure?

Reply:
We're invalidating the installer-generated kubeconfig and replacing it with a newly generated obe. The new one must be added and configured before the installer-generated one is invalidated. (The existing one can't be removed, it can be invalidated, though.)